data/json/mapgen/Glassblower.json

```diff
@@ -0,0 +1,147 @@
+[
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": [ "craft_shop" ],
+    "weight": 200,
+    "object": {
+      "fill_ter": "t_floor",
+      "rows": [
+        "________ss4;>;;>;;2>;;;;",
+        "________ss4>;>3;>;;7;;4;",
+        "_______ss6;>;;>72;;;446;",
+        "_______s%%--%rr%--%-%%;;",
+        "_______s%...........E%;;",
+        "_______sG.n.n.n.n.P%%%6;",
+        "______s5w.n.n.n.n.B%S%;;",
+        "______ss=.n.n.n.n..=.%57",
+        "______s5w.........R%t%1;",
+        "______s4GEEcccC...P%v%4;",
+        "______s;%....T....E%;4;6",
+        "______s4%BBB....PEE%;;;;",
+        "______s4%%%%%=%%xx%%;;4;",
+        "______s;5>651>11;55;;1;7",
+        "_____s43>;>41>1;4;;2;6;;",
+        "_____s4||||||=|||x||;;4;",
+        "_____s4|L??%***qNmY|>;;;",
+        "_____ss=***=*****#*|;2>;",
+        "______;|fLm%**qqq**|;;>;",
+        "______4|||v|Q*****m|>3;;",
+        "______;>2;5|Q*Qqq*Q|;64;",
+        "_______;u;H|Q*Q**qQ|2>;;",
+        "________;>H||r|G|w||1;4;",
+        "______UU7;;>;6>;;7;;;>;;"
+      ],
+      "terrain": {
+        ".": "t_floor",
+        "s": "t_sidewalk",
+        "u": "t_dirt",
+        "_": "t_pavement",
+        "U": "t_pavement",
+        ";": "t_grass",
+        ">": "t_dirt",
+        "H": "t_grass",
+        "1": "t_shrub_hydrangea",
+        "2": "t_tree_walnut",
+        "3": "t_tree_apple",
+        "4": "t_underbrush",
+        "5": "t_grass",
+        "6": "t_grass",
+        "7": "t_grass",
+        "-": "t_wall_glass",
+        "|": "t_rock_wall",
+        "%": "t_wall_wood",
+        "+": "t_door_glass_c",
+        "=": "t_door_c",
+        "G": "t_window_stained_green",
+        "w": "t_window_stained_blue",
+        "r": "t_window_stained_red",
+        "x": "t_window_no_curtains",
+        "v": "t_window_domestic",
+        "c": "t_floor",
+        "C": "t_console_broken",
+        "P": "t_floor",
+        "T": "t_floor",
+        "B": "t_floor",
+        "n": "t_floor",
+        "R": "t_floor",
```

Comment:
Your fill_ter is t_floor, so you don't really need to specify what's going to be underneath PTBnR